### PR TITLE
Add page from the conference chair

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: pages/from-the-conference-chair.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+I'd like to thank our American friends, [Naomi Ceder](https://twitter.com/NaomiCeder), [Van Lindberg](https://twitter.com/VanL), and [Carol Willing](https://twitter.com/WillingCarol), for advice before and during the conference, and to Van and Carol for helping with our AGMs on Saturday.
+
+The conference couldn't happen without the generous support of our sponsors.  There's a full list at the bottom of this page.  I hope you all found sponsoring the conference to be useful.  To the wider community: many of our sponsors are recruiting, so if you're looking for work, get in touch with them!
+
+The [Raspberry Pi Foundation](https://www.raspberrypi.org/) have been supporting the education track at PyCon UK for a number of years.  This year, they ran a [Picademy](https://www.raspberrypi.org/training/picademy/) for teachers on Thursday and Friday, and coordinated Saturday's young coders' day.  The whole team worked incredibly hard!  In particular, I'd like to thank [Carrie Anne Philbin](https://twitter.com/MissPhilbin), James Robinson, and [Ben Nutall](https://twitter.com/ben_nuttall) who were also involved before the conference.
```
Comment (alexwlchan) [inline pages/from-the-conference-chair.md]: “Nuttall” has two “t”s.

File: pages/from-the-conference-chair.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+[Charlie Denton](https://twitter.com/meshy) and [Cory Benfield](https://twitter.com/lukasaoz) were involved early on in the planning but were unable to make the conference.  You were missed!  (And the staff at [Pieminister](http://www.pieminister.co.uk/restaurants/cardiff/) want to know what to do with the pigs in blankets they'd set aside for you.)
+
+[Chloe Parkes](https://twitter.com/MissMaximas) responded to enquiry emails before the conference, and then was everywhere during the conference, picking up all the jobs that needed doing, as well as co-ordinating the reception desk.  She also gave an inspiring lightning talk.
+
+[Daniele Procida](https://twitter.com/evildmp) oversaw programme selection, looked after bookings with all the external suppliers, handled the morning announcements, and lent us his printer.
+
+[George Hickman](https://twitter.com/ghickhman) greased the wheels of the conference's techincal infrastructure and set up the conference Slack.
```
Comment (alexwlchan) [inline pages/from-the-conference-chair.md]: “techincal”?

File: pages/from-the-conference-chair.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+[Tim Golden](https://twitter.com/tjguk) helped with the financial assistance administration, offered lots of wisdom, and wrote the wonderful [Zen of PyCon UK](http://2017.pyconuk.org/zen-of-pycon-uk/).
+
+Tim Vivian-Griffiths co-ordinated the PyData track at the conference.
+
+Tim Williams has once again been brilliant at co-ordinating the filming of talks at the conference, and has been processing them and getting them [onto YouTube](https://www.youtube.com/channel/UChA9XP_feY1-1oSy2L7acog) in record time.
+
+[Vincent Knight](https://twitter.com/drvinceknight) was our Chief Mathematics Officer, and helped with scheduling.
+
+So thanks to all the comittee members.  I've really enjoyed working with you.
```
Comment (alexwlchan) [inline pages/from-the-conference-chair.md]: “committee” has two “m”s.

**Review by alexwlchan — CHANGES_REQUESTED** [review]
Never one to break tradition, I proofread this page as well. A couple of typos, see inline.

Also, you need to link it from somewhere, or it won’t appear on the site.

**Review by alexwlchan — APPROVED** [review]
Okay, I’ve read this three times. If there are still typos, I’m past the point of being able to see them. Happy for this to be merged as-is.

**Comment by inglesp** [comment]
Thanks @alexwlchan!  Let's add a spell-checker to Travis for next year...

**Pull request merged**